_opensearch/install/rpm.md

```diff
@@ -0,0 +1,149 @@
+   ```
+
+   ```bash
+   sudo curl -SL https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/{{site.opensearch_version}}/opensearch-dashboards-{{site.opensearch_version}}.repo -o /etc/yum.repos.d/{{site.opensearch_version}}.repo
+   ```
+
+2. Navigate to your YUM repository directory:
+
+   ```bash
+   cd /etc/yum.repos.d/
+   ```
+
+3. Clean your YUM cache, to ensure a smooth installation:
+
+   ```bash
+   sudo yum clean all
+   ```
+
+4. With the repository file downloaded, list all available versions of OpenSearch:
+
+   ```bash
+   sudo yum list | grep opensearch
+   ```
+
+5. Chose the version of OpenSearch you want to install: 
+
+   ```bash
+   sudo yum install opensearch
+   sudo yum install opensearch-dashboards
+   ```
+
+   Unless otherwise indicated, the highest minor version of OpenSearch installs.
+
+   To install a specific version of OpenSearch
+
+   ```bash
+   sudo yum install 'opensearch-{{site.opensearch_version}}'
+   ```
+
+6. During installation, the installer stops to see if the GPG key matches the OpenSearch project. Verify that the `Fingerprint` matches the following:
+
+   ```bash
+   Fingerprint: c5b7 4989 65ef d1c2 924b a9d5 39d3 1987 9310 d3fc
+   ```
+
+   If correct, enter `yes` or `y`. The OpenSearch installation continues.
+  
+   Once complete, you can run OpenSearch inside your distribution. 
+
+## Run OpenSearch
+
+1. Run OpenSearch and OpenSearch Dashboards using `systemctl`.
+
+   ```bash
+   sudo systemctl start opensearch.service
+   sudo systemctl start opensearch-dashboards.service
+   ```
+
+  If you configure the security plugin for production use (or disable it), you can run OpenSearch using `./bin/opensearch`.
+
+2. Send requests to the server to verify that OpenSearch is running:
+
+   ```bash
+   curl -XGET https://localhost:9200 -u 'admin:admin' --insecure
+   curl -XGET https://localhost:9200/_cat/config?v -u 'admin:admin' --insecure
+   ```
+
+3. To stop running OpenSearch, enter:
+
+   ```bash
+   sudo systemctl stop opensearch.service
+   sudo systemctl stop opensearch-dashboards.service
+   ```
+
+
+## (Optional) Set up Performance Analyzer
+
+When enabled, the Performance Analyzer plugin collects data related to the performance of your OpenSearch instance. To start the Performance Analyzer plugin, enter:
+
+```bash
+sudo systemctl start opensearch-performance-analyzer.service
+```
+
+To stop the Performance Analyzer, enter:
+
+```bash
+sudo systemctl stop opensearch-performance-analyzer.service
+```
+
+## Upgrade RPM
+
+You can upgrade your RPM OpenSearch instance both manually and through YUM. 
+
+
+### Manual 
+
+Download the new version of OpenSearch you want to use, and then use `rmp -Uvh` to upgrade.
+
+### YUM
+
+To upgrade to the latest version of OpenSearch with YUM, use `yum upgrade`. You can also upgrade to a specific OpenSearch version by using `yum upgrade opensearch-<version-number>`.
```

Comment:
yum upgrade forces the removal of obsolete packages, while yum update may or may not also do this. The removal of obsolete packages can be risky, as it may remove packages that you use.

This makes yum update the safer option.

Thanks.